src/agents/customer_group.jl

```diff
@@ -345,47 +357,47 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         ParamArray(
             "Constant",
             (model_data.index_h,),
-            [0.0, 0.0, 0.0]; 
+            vcat(fill(0.0, 66)),
             description = "Constant in green power uptake function (regression parameter)",
         ),
         ParamArray(
             "GreenPowerPrice_coefficient",
             (model_data.index_h,),
-            [0.0, -0.55, -0.55];
+            vcat(fill(0.0, 64), [-0.55, -0.55]),
```

Comment:
To not hard-code this, need to use the index_h_to_sector mapping (which hasn't yet been pulled into HEMData, but should be). Ditto for the non-zero values below.